bindings/node/index.d.ts

```diff
@@ -203,7 +203,13 @@ export interface ClientEncryptionOptions {
   /**
    * TLS options for kms providers to use.
    */
-  tlsOptions?: ClientEncryptionTLSOptions;
+  tlsOptions?: {
+    aws?: ClientEncryptionTLSOptions;
+    local?: ClientEncryptionTLSOptions;
+    azure?: ClientEncryptionTLSOptions;
+    gcp?: ClientEncryptionTLSOptions;
+    kmip?: ClientEncryptionTLSOptions;
+  }
```

Comment:
If the expectation is that this list is always supposed to match the `KMSProviders` keys, and that we likely won’t need special options for individual ones, this could also be:

```suggestion
  tlsOptions?: { [kms in keyof KMSProviders]?: ClientEncryptionTLSOptions };
```

(or maybe

```suggestion
  tlsOptions?: { [kms in Exclude<keyof KMSProviders, 'local'>]?: ClientEncryptionTLSOptions };
```

I’m not sure how much sense it makes to specify TLS options for `local` … :slightly_smiling_face: )

Reply:
The spec doesn't explicitly exclude local so I took the first suggestion.